package.json

```diff
@@ -0,0 +1,50 @@
+{
+   "name": "glimmer.js",
+   "version": "0.8.0",
```

Comment:
I was thinking we should synchronize versions here like we do with glimmer-vm since `@glimmer/application` and `@glimmer/component` are so coupled. They're on 0.7.2 and 0.6.0 respectively now so I think we'd need to bump to 0.8.0 if we want to sync versions.